packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -46,7 +49,14 @@ export async function loadConfig(cwd: string) {
 
   const options = await architectHost.getOptionsForTarget(buildTarget);
   if (!options) throw new Error("Not able to find options for build target.");
-  return options;
+
+  // options has to be of type ApplicationBuilderOptions when the builder is an application builder
+  const applicationBuilderOptions: ApplicationBuilderOptions = Object.assign(
```

Comment:
Can we cooerce the type without an object assign? also you can return this object immediately w/o the intermediary const

Reply:
added zod validation